src/fomosto/psgrn_pscmp.py

```diff
@@ -1545,6 +1540,11 @@ def init(store_dir, variant):
         modelling_code_id='psgrn_pscmp.%s' % variant,
         tabulated_phases=[])    # dummy list
 
+    def to_save_burger(val):
+        return literal(cake.write_nd_model_str(val, burgers_material=True))
+
+    config._ConfigTypeA__T.cls.earthmodel_1d.to_save = to_save_burger
```

Comment:
@emolch this might be a little bloody, digging too deeply in the guts. Yet `init` is called from `fomosto` CLI only.